client/components/overview/Overview.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import axios from 'axios';
+import Summary from './Summary.jsx';
+import StarDistribution from './StarDistribution.jsx';
+import LovedFor from './LovedFor.jsx';
+import exampleData from './../exampleData.js';
+import overview from './helperFunctions.jsx';
+
+class Overview extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      totalReviews : 1381,
+      neighborhood : 'SOMA',
+      overallRating : overview.overallRating(exampleData),
+      foodRating : overview.foodRating(exampleData),
+      serviceRating : overview.serviceRating(exampleData),
+      ambienceRating : overview.ambienceRating(exampleData),
+      valueRating : overview.valueRating(exampleData),
+      noiseLevel : overview.noiseLevel(overview.noise(exampleData)),
+      recommended : overview.recommended(exampleData)
+    }
+  }
+
+
+  render() {
+    axios.get('/reviews')
+      .then((response) => {
```

Comment:
If your  ES6 function only has one parameter, you do not need to put parentheses.